.github/workflows/report.yml

```diff
@@ -19,26 +19,27 @@ jobs:
           fetch-depth: 0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@1.2.0
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.6.1
 
       - name: Download Artifacts
         uses: actions/download-artifact@v4.2.1
         with:
           path: ./artifacts
 
       - name: Copy Artifacts into Root Folder
-        working-directory: ./artifacts
-        run: |
-          poetry run -- coverage combine --keep coverage-python3.10*/.coverage
-          # Errors during copying are ignored because they are checked in the next step
-          cp .coverage ../ || true
-          cp lint-python3.10/.lint.txt ../ || true
-          cp lint-python3.10/.lint.json ../ || true
-          cp security-python3.10/.security.json ../ || true
+        run: poetry run -- nox -s artifacts:copy -- artifacts
 
       - name: Validate Artifacts
         run: poetry run -- nox -s artifacts:validate
 
+      - name: Upload to sonar
+        env:
+          SONAR_TOKEN: "${{ secrets.SONAR_TOKEN }}"
+        run: poetry run -- nox -s sonar:check
+
+      # Maybe we need to disable reporting again
```

Comment:
Yeah, might be that `project:report` needs to be turned off. Think the sonar should be ok with this error though (would just make sure it's not part of the branch protections):
<img width="1928" height="280" alt="image" src="https://github.com/user-attachments/assets/d56ae263-9ec5-446f-9983-7762f67afbfe" />

Though you don't have the sonar section in the pyproject.toml yet. nor does it look like we have sonar project created yet. Would also suggest checking to see if the GitHub app & token have been added.

Reply:
With the sonar comments, that's only if you want to go that way already. If not, it's ok to comment out, as I don't think it was previously activated.

Reply:
looks like we've decided to turn this off for now. fair.